pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -17,6 +17,7 @@
 , clojure-lsp
 , alejandra
 , millet
+, plantuml
```

Comment:
```suggestion
, plantuml
```

I understand this will work. But I find this pattern for a long file odd.

Maybe `jebbs.plantuml = callPackage ./jebbs.plantuml { };` instead.

@jonringer opinions this?

Reply:
As it's been a while and no answer.
I think this package should be in it's own directory (jebbs.plantuml). Because of `plantuml`.